acceptance/2016nano/.acceptance/acceptance-cookbook/recipes/provision.rb

```diff
@@ -0,0 +1,41 @@
+run_tf "plan"
+
+run_tf "apply"
+
+ruby_block "install via winrm api" do
+  block do
+    # We do this hackery to ensure we load mixlib-install from source
+    install_command = "install -project angry-omnibus-toolchain -channel current -version 1.1.57"
```

Comment:
Just a heads up `angry-omnibus-toolchain 1.1.57` will eventually be reaped from the `current` channel. We should land a release in the `stable` channel (which is never reaped) before that is an issue.